pkg/types/gcp/validation/platform.go

```diff
@@ -104,8 +108,8 @@ func ValidatePlatform(p *gcp.Platform, fldPath *field.Path, ic *types.InstallCon
 	}
 
 	for i, license := range p.Licenses {
-		if validate.URIWithProtocol(license, "https") != nil {
-			allErrs = append(allErrs, field.Invalid(fldPath.Child("licenses").Index(i), license, "licenses must be URLs (https) only"))
+		if license != nestedVirtLicense {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("licenses").Index(i), license, "only accepted license is enable-vmx"))
```

Comment:
If you remove the creation of the image with the licenses in terraform, specifying any license in the install-config does nothing.

Reply:
Are you thinking we should just change the license field description and put out a warning if it is used?

Reply:
Yes, that's what I had in mind. We need to deprecate the field first, and then after a few releases we can remove.